src/compass/utils/stitching/stitch_burst.py

```diff
@@ -0,0 +1,404 @@
+import argparse
+import glob
+import json
+import os
+import time
+
+import isce3
+import journal
+import pandas as pd
```

Comment:
@vbrancat do you anticipate pandas being used outside burst stitching? I'm split between using pandas to make it super easy to select/filter bursts/dates and keeping dependencies light by writing some not-too-complex logic.

Reply:
@LiangJYu I would like to use `pandas` also in the geocoded and radar burst stack processors. I find it very handy in handling the bursts to process. Using `pandas` should be simple and fast enough to do not require writing any additional logic. Not sure if @hfattahi has a strong opinion on this matter.

Reply:
I agree with Virginia. We can hide the pandas dependency for SDS deliveries as they won't use the stack processor or stitcher.